snakemake/__init__.py

```diff
@@ -1122,8 +1147,9 @@ def get_argument_parser(profile=None):
             "Use at most N CPU cores/jobs in parallel. "
             "If N is omitted or 'all', the limit is set to the number of "
             "available CPU cores. "
-            "In case of cluster/cloud execution, this argument sets the number of "
-            "total cores used over all jobs (made available to rules via workflow.cores)."
+            "In case of cluster/cloud execution, this argument sets the maximum number "
+            "of cores requested from the cluster or cloud scheduler. "
+            "This number is available to rules via workflow.cores."
```

Comment:
Let us add a link to your nice explanation about the scopes in the docs here.

Reply:
How should I add the link? I can do a :ref:`tag` but that won't render in the CLI help (or at least I don't think it will). Or I can do a hard link, but then if the section moves the link won't update. Is there a better way?

Reply:
I think all we can do is a full URL to the stable docs page. Better than nothing.